content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -5,60 +5,96 @@ weight: 150
+
+The steps in this guide cover how to use an IAM user with long term programmatic credentials to use in O3DE. If you don't have IAM access keys configured, use the AWS Console to generate and download new access keys for an existing or new IAM user using the steps shown in the AWS reference guide for [Programmatic access](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
+
+If you want to use short-term credentials for working with AWS, please see setup information in the [AWSClientAuth Gem](/docs/user-guide/gems/reference/aws/aws-client-auth).
+
+Choose from the following options to set up a user's AWS credentials for use in O3DE. If you are using [named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) remember to set the profile in the project settings.
+
+### Option 1: Create a named profile using the AWS CLI
+O3DE recommends using the [AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) (CLI) (version 2) to manage the import and configuration of AWS credentials. If you have not configured credentials or a region on your computer, the easiest way to satisfy this requirement is to use the AWS `configure` command:
 
 ```cmd
 aws configure
 ```
 
-Using this command you can provide your AWS access key ID, secret access key, and default region when prompted.
-
-If you don't have access keys and secret keys, you must sign in as an IAM user, create an access key, and download credentials using the steps shown in the AWS Reference Guide for [Programmatic Access](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
+Using this command you can provide your AWS access key ID, secret access key, and default region manually when prompted. 
 
-You can easily imported the exported credentials using the AWS `import` sub-command (requires AWS CLI version 2):
+Alternatively, when you create new access keys for a user, you are given the option to download the keys as a CSV. You can then automatically import them using the AWS `import` CLI command (requires AWS CLI version 2):
 
 ```cmd
 aws configure import --csv file://credentials.csv
 ```
 
-For more information on using AWS CLI configure commands, see [Configuration and Credential File Settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) in the AWS CLI User Guide.
+This will create a named profile based on the name of the IAM user in your `credentials` file. 
+
+You can control which profile is used by default in the AWS CLI either by setting a ```[default]``` profile or through the use of the [AWS_DEFAULT](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html#using-profiles) environment variable.
```

Comment:
Whoops. I think we both agreed that you want to change the environment variable to AWS_PROFILE, to match the linked doc.
```suggestion
You can control which profile is used by default in the AWS CLI either by setting a ```[default]``` profile or through the use of the [AWS_PROFILE](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html#using-profiles) environment variable.
```

Reply:
Nice catch, thanks.